lib/primo/query.rb

```diff
@@ -138,40 +141,42 @@ def required_params_included?(params)
     end
 
     def field_is_known?(params)
-      field = params.fetch(:field)
+      field = params[:field].to_sym
```

Comment:
Is this switch away from fetch because you assume that the params will have been passed with a default value by the calling method?

Reply:
I think it's just because `params[:field].to_sym` is shorter than `params.fetch(:field).to_sym` I think either would work.

Reply:
OK.

Reply:
I went ahead and updated this bit to be more explicit so there is no failure caused by `nil.to_sym` exception.  Thanks for the suggestion!